src/main/java/core/basesyntax/Main.java

```diff
@@ -3,6 +3,6 @@
 /**
  * Feel free to remove this class and create your own.
  */
-public class HelloWorld {
+public class Main {
```

Comment:
You need to show how your FruitShop is working in the main method, [see common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop).
Also, where's the resources folder with a .csv file in it, so you can read from it